chingu-coders/Voyage2-Turtles-02
# Update styling

## Purpose
* Style features to look consistent throughout app
* Hide all content until fully loaded then fade everything in

## Approach/Fixes
* Used similar background/border code for similar features
* Used similar hover code for similar properties
* Used `visiblity: hidden` for html container and changed it to `visible` upon load
* Only show quicklinks when title/icon is hovered instead of entire body

## Testing
* Pull this code into your local machine
* Check to see that features look more similar
* When initially loading check to see if the hidden html is visible or not (color picker, page speed, validators, etc.)

## Issues
* Quicklinks not adding new link right away (shows up when page is refreshed; see console for errors)
